package.json

```diff
@@ -17,21 +17,20 @@
     "test-functional-windows": "tap -R spec --timeout=300 ./test/*.test.ts -g \"^(?!.*symlink)\"",
     "test-system-windows": "tap -R spec --timeout=300 ./test/system/*.test.ts -g \"^(?!.*prometheus)\"",
     "test-windows": "npm run test-functional-windows && npm run test-system-windows",
-    "semantic-release": "semantic-release",
```

Comment:
why is semantic release removed?

Reply:
The proper style, as taught by @lili2311 , is to use "npx semantic-release" in the .travis.yml, which automatically uses the latest and the greatest version.